src/index.ts

```diff
@@ -18,6 +18,11 @@ import { BCS, HexString, TxnBuilderTypes, Types } from 'aptos';
 import { txnAuthenticatorFromAccountAuthenticator } from './normalization';
 
 type ICAccount = Awaited<ReturnType<ICDappClient['getConnectedAccounts']>>[0];
+const ChainIds: Record<NetworkName, string> = {
+  mainnet: "0",
```

Comment:
I think mainnet is 1 and testnet is 2?

Reply:
You are correct! My brain just went right to 0 based indexing :joy: